cmake/FANSConfig.cmake.in

```diff
@@ -18,7 +18,6 @@ if (NOT HDF5_C_IS_PARALLEL)
     message(FATAL_ERROR "Parallel HDF5 implementation (mpi) required but not found!")
 endif()
 find_dependency(Eigen3)
-find_dependency(OpenMP)
```

Comment:
This change seems to be unrelated to this pull request. Please revert.

Reply:
this is part of the remove OpenMP dependency PR. I merged it into this PR to make it work temporarily.